cpp/example_code/s3/CMakeLists.txt

```diff
@@ -17,93 +17,87 @@ set(CMAKE_CXX_STANDARD 11)
 set(WINDOWS_BUILD ${MSVC})
 
 # Set the location of where Windows can find the installed libraries of the SDK.
-if(WINDOWS_BUILD)
+if (WINDOWS_BUILD)
     string(REPLACE ";" "/aws-cpp-sdk-all;" SYSTEM_MODULE_PATH "${CMAKE_SYSTEM_PREFIX_PATH}/aws-cpp-sdk-all")
     list(APPEND CMAKE_PREFIX_PATH ${SYSTEM_MODULE_PATH})
-endif()
+endif ()
 
 # CURL package is optional to test pre-signed url code.
 find_package(CURL)
 
-if (NOT CURL_FOUND) 
-    unset(CURL_INCLUDE_DIRS) 
-    unset(CURL_LIBRARIES) 
+if (NOT CURL_FOUND)
+    unset(CURL_INCLUDE_DIRS)
+    unset(CURL_LIBRARIES)
     unset(CURL_LIBRARY)
-endif()
+endif ()
 
 
 # Find the AWS SDK for C++ package.
 find_package(AWSSDK REQUIRED COMPONENTS ${SERVICE_COMPONENTS})
 
-if(WINDOWS_BUILD AND AWSSDK_INSTALL_AS_SHARED_LIBS)
-     # Copy relevant AWS SDK for C++ libraries into the current binary directory for running and debugging.
+if (WINDOWS_BUILD AND AWSSDK_INSTALL_AS_SHARED_LIBS)
+    # Copy relevant AWS SDK for C++ libraries into the current binary directory for running and debugging.
 
-     # set(BIN_SUB_DIR "/Debug") # If you are building from the command line, you may need to uncomment this 
-                                    # and set the proper subdirectory to the executables' location.
+    # set(BIN_SUB_DIR "/Debug") # If you are building from the command line, you may need to uncomment this
+    # and set the proper subdirectory to the executables' location.
```

Comment:
Is `executable` single or plural here? Singular would be "executable's location" and plural would be "executables' location"